news/7768.feature

```diff
@@ -0,0 +1 @@
+Add logger alert for when 'wheel' is not installed.
```

Comment:
```
Indicate when wheel building is skipped, due to lack of the ``wheel`` package.
```

Reply:
Updated in https://github.com/pypa/pip/pull/7768/commits/a39a04242fb24b931c76677a16a6ec8194138753 😄